src/bridge.h

```diff
@@ -37,6 +37,29 @@ constexpr uint SPRITES_PER_BRIDGE_PIECE = 32; ///< Number of sprites there are p
 
 typedef uint BridgeType; ///< Bridge spec number.
 
+/**
+ * Actions that can be performed when the vehicle enters the depot.
+ */
+enum BridgePiecePillarFlags {
+	BPPF_CORNER_W        = 1 << 0,
+	BPPF_CORNER_S        = 1 << 1,
+	BPPF_CORNER_E        = 1 << 2,
+	BPPF_CORNER_N        = 1 << 3,
+	BPPF_ALL_CORNERS     = 0xF,
+	BPPF_EDGE_NE         = 1 << 4,
+	BPPF_EDGE_SE         = 1 << 5,
+	BPPF_EDGE_SW         = 1 << 6,
+	BPPF_EDGE_NW         = 1 << 7,
+};
+DECLARE_ENUM_AS_BIT_SET(BridgePiecePillarFlags)
```

Comment:
Please use `enum class ... : uint8_t` and `EnumBitSet<>` to define the bitset flags. Should be plenty of examples around the code to show how these are used.